.github/workflows/sync-strings.yml

```
@@ -0,0 +1,45 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+name: "Sync Strings"
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+
+jobs:
+  main:
+    name: "Sync Strings"
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Discover Focus Beta Version"
+        id: focus-android-beta-version
+        uses: mozilla-mobile/fenix-beta-version@3.0.0
+      - name: "Skip non-beta versions"
+        uses: andymckay/cancel-action@0.2
+        if: ${{ steps.focus-android-beta-version.outputs.beta_version == '' }}
+      - name: "Checkout Main Branch"
+        uses: actions/checkout@v2
+        with:
+          path: main
+          ref: main
+      - name: "Checkout Beta Branch"
+        uses: actions/checkout@v2
+        with:
+          path: beta
+          ref: "releases_v${{ steps.focus-android-beta-version.outputs.beta_version }}.0.0"
+      - name: "Sync Strings"
+        uses: mozilla-mobile/sync-strings-action@1.0.1
+        with:
+          src: main
+          dst: beta
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          path: beta
+          branch: automation/sync-strings-${{ steps.focus-android-beta-version.outputs.beta_version }}
+          title: "Sync Strings from main to releases_${{steps.focus-android-beta-version.outputs.beta_version}}.0"
+          body: "This (automated) PR syncs strings from `main` to `releases_${{steps.focus-android-beta-version.outputs.focus-android-beta-version}}.0.0`"
```

One small detail: The output here should be just "beta_version" as above. 
And as a small style details spaces before and after accolades could be added also for consistency with the above code.
```suggestion
          body: "This (automated) PR syncs strings from `main` to `releases_${{ steps.focus-android-beta-version.outputs.beta_version }}.0.0`"
```